src/Razor/src/Microsoft.AspNetCore.Razor.Common/Microsoft.AspNetCore.Razor.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework);netstandard2.0;$(DefaultNetFxTargetFramework)</TargetFrameworks>
```

Comment:
Nit: Should we have a `<AllTFMs>` property so we don't copy this everywhere?

Reply:
I thought about it but decided against it for a couple of reasons:

1. It'd be unclear which frameworks (.NET Framework .NET Standard, .NET) are being targeted without looking up an MSBuild property. I prefer seeing which frameworks a project targets by inspecting the project file rather than hiding it elsewhere. I'm totally comfortable with hiding the versions though.
2. It doesn't add much value. We're going to ultimately be removing `netstandard2.0` in the tooling layer, once it's no longer needed for Roslyn OOP. When that happens, I'd prefer updating the affected projects rather than updating an MSBuild property and not knowing what projects are affected.
3. Existing projects get out of date. If `$(AllTFMs)` were modified at some point to remove `netstandard2.0`, we'd have to search through all projects to see which ones should _now_ use `$(AllTFMs)` because they match its current value. Or, we end up with projects being defined inconsistently which limits the value of having an MSBuild property in the first place.

Reply:
It could make sense to have MSBuild properties that describe the platform that is being targeted, assuming they're all different. For example, `<TargetFrameworks>$(VSMac);$(OmniSharp);$(VS);$(RoslynOO);$(RZLS)</TargetFrameworks>`. However, I'm not sure it buys us all that much. We really don't have _that_ many projects. :smile: